std-bits/table/src/main/java/org/enso/table/parsing/problems/ProblemAggregator.java

```diff
@@ -13,6 +13,12 @@ public class ProblemAggregator {
 
   private final List<String> invalidFormatCells = new ArrayList<>();
   private final List<String> leadingZerosCells = new ArrayList<>();
+  private int mismatchedQuotes = 0;
+  private final String relatedColumn;
```

Comment:
as it's the name of the column lets call it `columnName`

Reply:
Maybe `relatedColumnName` then?